components/calendar/calendar.js

```diff
@@ -307,7 +308,7 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 		super.updated(changedProperties);
 
 		changedProperties.forEach((oldVal, prop) => {
-			if (prop === '_shownMonth') {
+			if (prop === '_shownMonth' && this._keyboardTriggeredMonthChange) {
```

Comment:
Did some rearranging with `_keyboardTriggeredMonthChange` just so that it wasn't necessary to set it to true before every call to `_focusDateAddFocus`